EIPS/eip-6093.md

```diff
@@ -0,0 +1,376 @@
+---
+eip: 6093
+title: Custom errors for ERC tokens
+description: List of Solidity custom errors covering standard token implementations (EIP-20, EIP-721 and EIP-1155)
```

Comment:
```suggestion
description: Lists custom errors for common token implementations
```
This can't contain the word "standard", and is slightly too long.

Reply:
I agree with the suggestion, but same comments as [here](https://github.com/ethereum/EIPs/pull/6093#discussion_r1042227787) for the word `common`.

I'll update if I find a better way around.